exercises/simple-linked-list/src/simple_linked_list.app.src

```diff
@@ -0,0 +1,9 @@
+{application, simple_linked_list,
+  [{description, "exercism.io - simple-linked-list"},
+    {vsn, "0.0.1"},
```

Comment:
As we do not have canonical test data, could you please set the `vsn` to 0.0.0 as explained in #414?